src/www/ui/admin-license-file.php

```diff
@@ -352,7 +352,21 @@ private function isShortnameBlocked($rfId,$shortname,$text)
     $check_count = $this->dbManager->getSingleRow($sql,array($rfId,$shortname,$text),__METHOD__.'.countLicensesByNomos');
     return (0 < $check_count['count']);
   }
-  
+
+
+  /** @brief check if shortname is changed */
+  private function isShortNameExists($rfId, $shortname)
+  {
+    $sql = "SELECT rf_shortname FROM license_ref WHERE rf_pk=$1";
+    $row = $this->dbManager->getSingleRow($sql,array($rfId),__METHOD__.'.DoNotChnageShortName');
+    if ($row['rf_shortname'] === $shortname) {
```

Comment:
Please consider case insensitive comparison as right-hand value is coming from the UI form.